misc/images/materialized/README.md

```diff
@@ -19,13 +19,13 @@ The Materialize Emulator is an all-in-one Docker image available on Docker Hub,
 To launch the Docker container:
 
 ```
-docker run -v mzdata:/mzdata -p 6875:6875 -p 6876:6876 materialize/materialized
+docker run -d -p 6875:6875 -p 6876:6876 materialize/materialized:latest
```

Comment:
```suggestion
docker pull materialize/materialized:latest
docker run -d -p 6875:6875 -p 6876:6876 materialize/materialized:latest
```

Reply:
We can't easily keep the version up to date here, so having a `docker pull` before will ensure that `:latest` is updated.